spec/.rubocop.yml

```diff
@@ -5,3 +5,6 @@ StringLiterals:
   EnforcedStyle: double_quotes
   Exclude:
     - './spec_helper.rb'
+
+Metrics/BlockLength:
+  Enabled: false
```

Comment:
For RSpec files, almost the entire file is considered one block by Rubocop.  So, we quickly run over the maximum of 25 lines when writing specs.  Instead of setting a `Max` and having to adjust it every time we add tests, I decided to disable the rule.  Another option, is set it to a really high number, but probably not much better then disabling it.